lib/languageFeatures/completion.ts

```diff
@@ -140,7 +136,9 @@ export function attachOnCompletion() {
     for (const pkg of linter.packages) {
       const ieee = pkg.parent.file.match(/ieee/i) !== null;
       for (const obj of pkg.getRoot().objectList) {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
we actually have an type checking function for this now. implementsHasName or sth. Would be better then ignoring the rule.